playbooks/common/openshift-cluster/additional_config.yml

```diff
@@ -20,6 +20,8 @@
     when: openshift.common.install_examples | bool
   - role: openshift_cluster_metrics
     when: openshift.common.use_cluster_metrics | bool
+  - role: openshift_cluster_metrics_hawkular 
+    when: openshift.common.use_cluster_metrics_hawkular | bool and not openshift.common.use_cluster_metrics | bool
```

Comment:
This should be moved down to the "Create persistent volumes and create hosted services" play.

I'd say the role should also be named openshift_metrics as well.